src/main/java/net/sf/jabref/openoffice/CitationManager.java

```diff
@@ -58,13 +61,14 @@
     private static final Log LOGGER = LogFactory.getLog(CitationManager.class);
 
 
-    public CitationManager(final JabRefFrame frame, OOBibBase ooBase) throws Exception {
+    public CitationManager(final JabRefFrame frame, OOBibBase ooBase)
+            throws NoSuchElementException, WrappedTargetException, UnknownPropertyException {
         diag = new JDialog(frame, Localization.lang("Manage citations"), true);
         this.ooBase = ooBase;
 
         list = new BasicEventList<>();
         XNameAccess nameAccess = ooBase.getReferenceMarks();
-        String[] names = ooBase.getJabRefReferenceMarks(nameAccess);
+        java.util.List<String> names = ooBase.getJabRefReferenceMarks(nameAccess);
```

Comment:
Is there a reason to the fully qualified class name here? Why not add `import java.util.List;` to the class header? Is there a conflict with `java.awt.List`?